File: app/src/main/java/no/nav/tilbakemeldingsmottak/exceptions/ClientErrorNotFoundException.java
package no.nav.tilbakemeldingsmottak.exceptions;

// Errors som skal gi 404 respons
public class ClientErrorNotFoundException extends AbstractGeneralException {
    public ClientErrorNotFoundException(String message, ErrorCode errorCode) {
        super(message, errorCode);
    }

    public ClientErrorNotFoundException(String message, Throwable cause, ErrorCode errorCode) {
        super(message, cause, errorCode);
    }

}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/exceptions/ClientErrorNotFoundException.kt
package no.nav.tilbakemeldingsmottak.exceptions

// Errors som skal gi 404 respons
open class ClientErrorNotFoundException(
    override val message: String,
    override val cause: Throwable? = null,
    val errorCode: ErrorCode = ErrorCode.GENERAL_ERROR
) : RuntimeException(message)